test_move_script/Move.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = 'run_script'
+version = '1.0.0'
+
+[dependencies.AptosFramework]
+git = 'https://github.com/aptos-labs/aptos-core.git'
+rev = 'mainnet'
```

Comment:
Might be worth calling out in the readme that depending on what you're testing with, you might want to set this to devnet / testnet instead.